terraform/modules/gce/network/network/network.tf

```diff
@@ -0,0 +1,17 @@
+#--------------------------------------------------------------
+# This module creates all resources necessary for a Network
+#--------------------------------------------------------------
+
+variable "name"                    { default = "default_network" }
+variable "auto_create_subnetworks" { default = "true" }
+
+resource "google_compute_network" "network" {
+  name =                  = "${var.name}"
+  auto_create_subnetworks = "${var.auto_create_subnetworks}"
```

Comment:
Does this work in addition to the `private_subnet`'s you're also creating? Curious how they play together, I think I remember having issues getting this to work appropriately.